habitat/default.toml

```diff
@@ -1,11 +1,28 @@
-# The Chef run interval in seconds
-interval = 10 
+[chef-client]
```

Comment:
Should we be using underscores here? Would like to get consistency about this across core/chef plans. Not a blocker, but the other variables use underscores in this file.